bigtable-dataflow-parent/bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableConfiguration.java

```diff
@@ -253,6 +253,9 @@ public BigtableOptions toBigtableOptions() throws IOException {
   public Configuration toHBaseConfig() {
     Configuration config = new Configuration(false);
 
+    // Dataflow should use a different endpoint for data operations than online traffic.
+    config.set(BigtableOptionsFactory.BIGTABLE_HOST_KEY, "batch-bigtable.googleapis.com");
```

Comment:
Should this be configurable like "<google.bigtable.endpoint.host>bigtable.googleapis.com</google.bigtable.endpoint.host>"